examples/with-segment-analytics/hooks/useSegment.ts

```diff
@@ -0,0 +1,9 @@
+import { AnalyticsBrowser } from '@segment/analytics-next'
+
+const analytics = AnalyticsBrowser.load({
+  writeKey: process.env.NEXT_PUBLIC_SEGMENT_WRITE_KEY!
+})
+
+export function useSegment() {
```

Comment:
No need to use a hook -- just export analytics here!

Reply:
I'm honestly not following you. analytics needs to be called with the load method and writeKey, then  for pages with track events it needs to be imported to access the track method.

Are you saying I should rename useSegment and move it into a lib directory?

Reply:
I mean just export the analytics instance you instantiated on the first line and call it directly. There's no need to have a useSegment hook to "capture" that variable, it's just extra boilerplate.